scipy/spatial/transform/_rotation.pyx

```diff
@@ -1500,7 +1766,7 @@ cdef class Rotation:
         (2, 3)
 
         """
-        
+
```

Comment:
Maybe you can after all reapply your changes, but without these space corrections?

Reply:
Removed the space corrections, should be a bit easier to read now